ibis/backends/duckdb/tests/test_client.py

```diff
@@ -420,3 +420,26 @@ def test_memtable_doesnt_leak(con, monkeypatch):
     df = ibis.memtable({"a": [1, 2, 3]}, name=name).execute()
     assert name not in con.list_tables()
     assert len(df) == 3
+
+
+def test_pyarrow_batches_chunk_size(con):  # 10443
+    import numpy as np
+
+    t = ibis.memtable(
+        {
+            "id": np.arange(500000),
+            "name": np.random.choice(["Alice", "Bob", "Carol", "Dave"], size=500000),
+            "age": np.random.randint(20, 70, size=500000),
+        }
+    )
+    batches = t.to_pyarrow_batches(chunk_size=100_000)
```

Comment:
```suggestion
    batches = con.to_pyarrow_batches(t, chunk_size=100_000)
```

You need to use `con` here, otherwise you'll implicitly set the default backend, which is what's currently causing CI to fail.

Reply:
I'll absolutely change it! When does using the table implicitly set the default backend?

Reply:
Whenever you don't explicitly use a backend instance like `con`. _Something_ has to do the compute, and if you haven't specified that thing explicitly by constructing a backend instance and then using it, or by calling `ibis.set_backend(<some backend instance>)` then Ibis is going to pick the default backend (DuckDB).